src/node/util.ts

```diff
@@ -315,24 +326,25 @@ type TTYStream = (ReadableStream | WritableStream) & {
   getColorDepth(): number;
 };
 
-function isTTYStream(
-  stream: ReadableStream | WritableStream
-): stream is TTYStream {
+function isTTYStream(stream: unknown): stream is TTYStream {
   return (
+    stream != null &&
     typeof stream === 'object' &&
     'isTTY' in stream &&
     !!stream.isTTY &&
     typeof (stream as TTYStream).getColorDepth === 'function'
   );
 }
 
-const stdoutPlaceholder = Object.create(null);
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+const stdoutPlaceholder: StyleTextOptions['stream'] = Object.create(null);
 export function styleText(
   format: string | string[],
-  text: string,
+  text: unknown,
   { validateStream = true, stream = stdoutPlaceholder }: StyleTextOptions = {}
 ): string {
   validateString(text, 'text');
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-boolean-literal-compare
   if (validateStream !== true)
```

Comment:
If it's really unnecessary, can't we just fix the line? Otherwise if it is necessary, we should say why.

Reply:
This is from Node.js source code probably. I prefer not to update the actual implementation